config/environments/production.rb

```diff
@@ -18,7 +18,7 @@
 
   # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
-  # config.require_master_key = true
+  config.require_master_key = true
```

Comment:
This is the wrong way. Production needs to rely on ENV vars.

Reply:
This must have accidentally been changed when I rebased - will flip it back!